### ISSUE TITLE
Determine and fix any issues around VIC and Harbor with wildcard certificate

### ISSUE THREAD

**Issue by sflxn** [vmware/vic#4266]
As a user of VIC and Harbor, I should be able to use a wildcard certificate with Harbor and VIC.

Acceptance Criteria:  Robot scripts that verifies wildcard certificates works.

---

We need to first determine if this is an issue.  If it is, we need to fix it.

**Comment by sflxn** [comment]
Not sure this is an issue.  Ian looked at an issue that involved wildcard certificates but was gated by another issue so we never determined if this is an issue.
